pyramid_oauthlib/__init__.py

```diff
@@ -18,9 +19,11 @@
     'password',
     'refresh_token',
     'response_type',
+    'response_mode',
```

Comment:
It looks like this is part of OIDC. Maybe we can, just for readability and ease of comparison against oauthlib, separate this into parts with a comment, like here: https://github.com/oauthlib/oauthlib/blob/v2.1.0/oauthlib/common.py#L396